AmpleOrganics/Blaze.vue
# update dependancies

updated the dependancies to their latest minors

## QA
run `yarn` to install new dependencies
checkout the docs to make sure everything runs properly using `yarn serve` and clicking around

make sure unit tests still pass

most notable change is Vue 2.6.x being used but in theory should have no breaking changes. old slot syntax is supported and not deprecated yet
https://medium.com/the-vue-point/vue-2-6-released-66aa6c8e785e

will make another pr in the future to change the slot syntax to be Vue 3 ready
